src/test/utils/index/sudo_index.js

```diff
@@ -0,0 +1,25 @@
+/* Copyright (C) 2016 NooBaa */
+'use strict';
+
```

Comment:
why this is a new file instead of rename?

Reply:
notice that the old sudo_index file appear to be deleted. I guess that because there were a lot of changes in the file. all of the requires were changed. github detected this to be a new file instead of a rename